_inc/connect-button.js

```diff
@@ -138,7 +138,7 @@ jQuery( document ).ready( function( $ ) {
 				},
 				success: function( data ) {
 					var siteData = JSON.parse( data.data );
-					jetpackConnectButton.isPaidPlan = ! siteData.plan.is_free;
+					jetpackConnectButton.isPaidPlan = ! siteData.plan.is_free || siteData.is_pending_plan;
```

Comment:
Can you reverse the order of the RHS? in it's current state it's easy to mis-read the logic if you don't know operator precedence by heart

Reply:
I've reversed the order as requested. 👍